google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -655,8 +655,8 @@ public static Builder newBuilder(String topicName) {
 
     // Meaningful defaults.
     static final long DEFAULT_ELEMENT_COUNT_THRESHOLD = 100L;
-    static final long DEFAULT_REQUEST_BYTES_THRESHOLD = 1000L; // 1 kB
-    static final Duration DEFAULT_DELAY_THRESHOLD = Duration.ofMillis(1);
+    static final long DEFAULT_REQUEST_BYTES_THRESHOLD = 1L * 1024L * 1024L; // 1 MB
+    static final Duration DEFAULT_DELAY_THRESHOLD = Duration.ofMillis(10);
```

Comment:
Upon reflection, I think we have to hold off on changing this value until v2.0 of the library. For users who are not hitting the batch size limit, this will represent a ~10x increase in publish latency if they are using the default settings. It will be hard for users to understand where that is coming from in the absence of client-side metrics.